solutions/0.5 first web/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title> index</title>
+        <link rel="stylesheet" href = "style.css">
```

Comment:
instead of `href = "style.css"`, do `href="style.css"` (remove spaces before and after `=`)